cypress/integration/immutableDatabaseTests/workflowDetails.ts

```diff
@@ -72,8 +72,7 @@ describe('Dockstore Workflow Details', () => {
     cy.get('[data-cy=dockstore-snapshot-unlocked]')
       .its('length')
       .should('be.gt', 0);
-        cy.get('[data-cy=dockstore-snapshot]')
-          .should('not.be.visible');
```

Comment:
:eyes:

Reply:
Elaborate

Reply:
The name of the test is blah blah blah "not see snapshot".
The not see snapshot part is commented out.